branchinfo.txt

```diff
@@ -4,5 +4,5 @@
 MAJOR_VERSION=1
 MINOR_VERSION=0
 PATCH_VERSION=0
-RELEASE_SUFFIX=rc2
+RELEASE_SUFFIX=rc2-preview1
```

Comment:
@piotrpMSFT Having some trouble updating this file, IIRC you did some git magic here, is there something special I need to do to update it?